### ISSUE TITLE
[HELP] Ubuntu 24.04 LTS installation

### ISSUE THREAD

**Issue by cos-gl** [thingsboard/thingsboard-gateway#1380]
**Describe the issue**
Trying to install deb package following instruction, raise an error about python3-cryptography package: needed pack version < 3.5 but system default is 41.0.7 in Ubuntu **24.04 LTS**

Thanks for your help!

**Configuration** Non relevant

**Error traceback:**
```
The following packages have unmet dependencies:
 python3-thingsboard-gateway : 
(...)
                               Depends: python3-cryptography (< 3.5) but 41.0.7-4build3 is to be installed
(...)
```

**Versions:**
 - OS: Ubuntu 24.04 (noble)
 - Thingsboard IoT Gateway version: 3.4.6
 - Python version: 3.12
 - Python3-cryptography: 41.0.7-4build3

**Comment by cos-gl** [comment]
Just to add some hints to someone how can put hands on...

The requirements.txt file seems that have been changed to exclude the python3-cryptograpy version check, al least this is what I understand looking here: https://github.com/thingsboard/thingsboard-gateway/commit/cb92f04d4be4de4e96a9e893727db596598b90f9
(last commit one month ago)

But the setup.py file, was not changed and continues to check for the package from version 3.4. to 3.5.
https://github.com/thingsboard/thingsboard-gateway/blob/783c7e1efe3763017186999e0fe410af63e1c322/setup.py#L56
(last commit 3 months ago)

Do I understand right? Is it possible to fix?
The installation using the .deb file off course doesn't work.
Thanks again!

**Comment by cos-gl** [comment]
Hi, no help for me? @samson0v  @imbeacon 
After a month I see the same problem and I'm in doubt which is the correct way to install TB Gateway (off course, without using docker). Thanks!

**Comment by cos-gl** [comment]
The dependency problem seems solved with the latest release of gateway: 3.5.1-1
